CondCore/HcalPlugins/plugins/HcalQIETypesPyWrapper.cc

```diff
@@ -0,0 +1,183 @@
+
+		ss << "Total HCAL containers: " << allContainers.size() << std::endl;
+
+		typedef std::pair<std::pair< std::vector<float>, std::vector<float> >, int> tPora;
+
+		std::vector<tPora> vec(allContainers.size());
+
+		std::vector<tPora>::iterator iMaz = vec.begin();
+
+		float sum = 0.0, average = 0.0, std_dev = 0.0, sqr_sum = 0.0;
+		int size = 0;
+
+		//Run trough all 8 detector containers:
+		for (iter = allContainers.begin(), iMaz = vec.begin(); iter != allContainers.end(); ++iter, ++iMaz){
+			ss << "---------------------------------------------" << std::endl;
+			ss << "Detector: " << (*iter).first << ";    Total values: "<< (*iter).second.size() << std::endl;
+			unsigned int j = 0;
+			iMaz->second = (*iter).second.size();
+			(iMaz->first).first = std::vector<float>(totalValues, 0.0);
+			(iMaz->first).second = std::vector<float>(totalValues, 0.0);
+			float capValue = 0.0;
+			//Run trough all values in container
+			for (contIter = (*iter).second.begin(); contIter != (*iter).second.end(); ++contIter){
+
+				//Run trough all values (8) in object:
+				for (unsigned int i = 0; i < totalValues; ++i){
+					capValue = (*contIter).getValue();
+					(iMaz->first).first[i] += capValue;
+					(iMaz->first).second[i]+= (capValue * capValue);
+					//ss << "[" << i << "] " << capValue << ", " << (iMaz->first).first[i] << ", " << (iMaz->first).second[i] << "; ";
+				}
+				//ss << std::endl;
+				++j;
+			}
+
+			size = (*iMaz).second;
+			for (unsigned int i = 0; i < totalValues; ++i){
+				sum = ((*iMaz).first).first[i];
+				sqr_sum = ((*iMaz).first).second[i];
+				average = sum/size;
+				std_dev = sqrt( (sqr_sum/size) - (average * average) );
+
+				ss  << "   HcalQIEType " << " :"<< std::endl;
+				ss	<< "          Average: " << average << "; "<< std::endl;
+				ss	<< "          Standart deviation: " << std_dev << "; " << std::endl;					
+
+			}	
+		}
+		//std::cout << ss.str();
+			return ss.str();
+	}
+
+	template<>
+	std::string PayLoadInspector<HcalQIETypes>::plot(std::string const & filename,
+		std::string const &,
+		std::vector<int> const&,
+		std::vector<float> const& ) const 
+	{
+
+		//how much values are in container
+		unsigned int numOfValues = 1;
+
+		//create object helper for making plots;
+		HcalQIETypesDataRepr datarepr(numOfValues, object().getAllContainers());
+
+		datarepr.nr = 0;
+		datarepr.id = 0;
+		datarepr.rootname.str("_HcalQIETyperootvalue_");
+		datarepr.plotname.str("HcalQIEType ");
+		datarepr.filename.str("");
+		datarepr.filename << filename << "";
+
+		typedef std::vector<TH2F> graphData;
+		std::vector< graphData > graphDataVec(numOfValues);
+		std::vector< graphData >::iterator imageIter;
+
+		//create images:
+		for (imageIter = graphDataVec.begin(); imageIter != graphDataVec.end(); ++imageIter){
+			//MAIN FUNCTION:
+			datarepr.fillOneGain((*imageIter));
+
+			++(datarepr.nr);
+			++(datarepr.id);
+		}
+		return filename;
+	}
+}
+PYTHON_WRAPPER(HcalQIETypes,HcalQIETypes);
```

Comment:
In that case will be also necessary to remove the last lines of
CondCore/HcalPlugins/plugins/BuildFile.xml